hr_employee_cost_history/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_hr_employee_timesheet_cost_user,access_hr_employee_timesheet_cost_user,model_hr_employee_timesheet_cost_history,base.group_user,1,0,0,0
+access_hr_employee_timesheet_cost_wizard_admin,access_hr_employee_timesheet_cost_wizard_admin,model_hr_employee_timesheet_cost_wizard,hr_timesheet.group_timesheet_manager,1,1,1,0
```

Comment:
issue: I don't think every DB user should be able to see other employees' cost history. That'd be a big problem in many companies.

[Upstream, HR users have R/W access to the `timesheet_cost` field](https://github.com/odoo/odoo/blob/a44d51b779bed1a04a46f7a08da3a0fd5581e4d5/addons/hr_timesheet/models/hr_employee.py#L11). We're just adding historical features over it, but IMHO we should keep the same permissions schema.

```suggestion
access_hr_employee_timesheet_cost_user,access_hr_employee_timesheet_cost_user,model_hr_employee_timesheet_cost_history,hr.group_hr_user,1,0,0,0
access_hr_employee_timesheet_cost_wizard_user,access_hr_employee_timesheet_cost_wizard_user,model_hr_employee_timesheet_cost_wizard,hr.group_hr_user,1,1,1,1
```

Reply:
Totally agree. Would you recommend adding a Form test for this type of permissions? Besides that, thanks for your suggestions.

Reply:
Yes, `Form` should trigger the permissions error (if there's one).